src/backend-dbus/actions.c

```diff
@@ -882,6 +888,8 @@ my_desktop_help (IndicatorSessionActions * self G_GNUC_UNUSED)
 {
   if (ayatana_common_utils_have_budgie_program ("yelp"))
     ayatana_common_utils_execute_command ("yelp help:gnome-user-guide");
+  else if (ayatana_common_utils_is_lomiri())
+    ayatana_common_utils_open_url("http://www.askubuntu.com");
```

Comment:
We probably don't want to send users of Lomiri to AskUbuntu. Maybe send to UBports forum (https://forums.ubports.com/) instead?